uint8 drive_command
uint8 move_select
float32 position
float32 velocity
float32 accel
float32 decel
float32 force
uint32 motion_type
uint32 outputs
